app/.gitignore

```diff
@@ -0,0 +1,2 @@
+venv
+__pycache__
```

Comment:
use **/__pycache__ to ignore __pycache__ directories on any level of the project cause they can arise at any module

Reply:
you can take a look at exemplary gitignore for Python
https://github.com/github/gitignore/blob/main/Python.gitignore